src/java/org/apache/cassandra/tools/nodetool/Status.java

```diff
@@ -72,26 +114,18 @@ public void execute(NodeProbe probe)
 
         Map<String, Float> ownerships = null;
         boolean hasEffectiveOwns = false;
-        try
-        {
+        try {
```

Comment:
keep these braces on new lines as they were before

Reply:
Okay I will keep the braces in new lines